pythonlib/src/mill/pythonlib/PythonModule.scala

```diff
@@ -22,9 +22,21 @@ trait PythonModule extends Module with TaskModule { outer =>
   def sources: T[Seq[PathRef]] = Task.Sources { millSourcePath / "src" }
 
   /**
-   * The script to run. This file may not exist if this module is only a library.
+   * The folders where the resource files for this module live.
    */
-  def script: T[PathRef] = Task.Source { millSourcePath / "src" / "main.py" }
+  def resources: T[Seq[PathRef]] = Task.Sources { millSourcePath / "resources" }
+
+  /**
+   * All Python source files in this module, recursively discovered from the source directories.
+   */
+  def allSourceFiles: T[Seq[PathRef]] = Task {
+    sources().flatMap(src => os.walk(src.path).filter(_.ext == "py").map(PathRef(_)))
+  }
+
+  /**
+   * The mainScript to run. This file may not exist if this module is only a library.
```

Comment:
It looks like you did a search-replace of `script` to `mainScript`. Any reason for this?

If you look at `man python3`, you'll see that they call the file to execute `script` everywhere. It's a detail of course, but I'd favor staying closer to nomenclature that is used in the python ecosystem itself.

Reply:
@lihaoyi Said 😅

Reply:
@jodersky BTW Sir Can you help me with this https://github.com/com-lihaoyi/mill/pull/4000#issuecomment-2495441017

Reply:
ok, that's fine by me. But be careful when replacing this term in the docs. Right now it says `the mainScript` to run`.

Reply:
FIXED!